lib/NanoPbComm/proto/gaggimate.proto

```
@@ -0,0 +1,200 @@
+syntax = "proto3";
+
+// Namespaced so the generated C types (gaggimate_Frame, gaggimate_Error, ...)
+// never collide with app-side structs of the same short name.
+package gaggimate;
```

Buf (1.69.0)

[error] Files with package "gaggimate" must be within a directory "gaggimate" relative to root but were in directory "lib/NanoPbComm/proto".

(PACKAGE_DIRECTORY_MATCH)